test/objects/accessmanagement/accessmanagement.go

```diff
@@ -50,3 +50,23 @@ func WithAccessRules(accessRules []v1alpha1.AccessRule) Opt {
 		am.Spec.AccessRules = accessRules
 	}
 }
+
+func WithLabels(kv ...string) Opt {
+	return func(am *v1alpha1.AccessManagement) {
+		if am.Labels == nil {
+			am.Labels = make(map[string]string)
+		}
+		if len(kv) == 0 {
+			return
+		}
+		if len(kv)&1 != 0 {
+			panic("expected even number of args")
+		}
+		for k := range am.Labels {
+			delete(am.Labels, k)
+		}
```

Comment:
What’s the reason for iterating through and deleting all the items instead of just creating a new map? I’m not opposed to it, I’m just trying to understand why you chose this approach.